shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/ddl/drop-view.xml

```diff
@@ -17,5 +17,5 @@
   -->
 
 <sql-cases>
-    <sql-case id="drop_view" value="DROP VIEW kinds" db-types="PostgreSQL" />
+    <sql-case id="drop_view" value="DROP VIEW kinds" db-types="PostgreSQL, SQLServer" />
 </sql-cases>
```

Comment:
@Icesource It's better to keep `SQLserver`, maybe we can use `xxxStatement` instead of `SQLServerxxxStatement` and modify visitor of these rule.

Reply:
@Icesource there, you delete test case for SQLServer, can you  tell me why？if it will fail, maybe relate to the definition of `statement` and visitor of these rules.

Reply:
```
<sql-cases>
-    <sql-case id="drop_view" value="DROP VIEW kinds" db-types="PostgreSQL" />
+    <sql-case id="drop_view" value="DROP VIEW kinds" db-types="PostgreSQL, SQLServer" />
</sql-cases>
```
@jingshanglu This shows that I added a test on my page. It turned out that this test was only performed for PostgreSQL. Since the delete statement is the same, I also added SQLServer here. I did not delete any tests and the tests did not fail. 
Maybe you read it wrong ?